doc/configuration/EDRConfiguration/EDR.md

```diff
@@ -8,17 +8,32 @@ The OGC API EDR service can be configured in a dataset configuration by adding t
 
 ```xml
```

Comment:
Can you add above that currently position and cube are supported?

Reply:
Done